.buildkite/scripts/pipelines/pull_request/pipeline.ts

```diff
@@ -68,6 +68,16 @@ const uploadPipeline = (pipelineContent: string | object) => {
       pipeline.push(getPipeline('.buildkite/pipelines/pull_request/security_solution.yml'));
     }
 
+    if (
+      (await doAnyChangesMatch([
+        /^x-pack\/plugins\/threat_intelligence/,
+        /^x-pack\/test\/threat_intelligence_cypress/,
```

Comment:
should we also add `security_solution/public/threat_intelligence` to the list?

Reply:
yeah, makes sense, thanks!

Reply:
In the future, we should consider adding other code paths for the things we pass from security_solution to our plugin through context